### ISSUE TITLE
QasmSimulator ignores configuration argument

### ISSUE THREAD

**Issue by ihincks** [Qiskit/qiskit-aer#835]
### What is the current behavior?

```
def __init__(self, configuration=None, provider=None):
        super().__init__(
            qasm_controller_execute,
            QasmBackendConfiguration.from_dict(self.DEFAULT_CONFIGURATION),
            provider=provider)
```

**Comment by aman3014** [comment]
I can fix this, if it hasn't already been done

**Comment by chriseclectic** [comment]
This will be fixed in #645/#486 where we include support for configurable backends.

**Comment by vvilpas** [comment]
Closing as #486 is already merged
